src/credentials/GroupDataProvider.h

```diff
@@ -28,6 +28,8 @@
 namespace chip {
 namespace Credentials {
 
+using GroupOperationalCredentials = Crypto::GroupOperationalCredentials;
```

Comment:
Please don't put using declarations not scoped to a specific class in headers.

Reply:
Removed